main/integration-tests/support/resources/projects/project.js

```diff
@@ -29,6 +29,11 @@ class Project extends Resource {
     if (_.isEmpty(parent)) throw Error('A parent resource was not provided to resource type [project]');
   }
 
+  async cleanup() {
+    if (this.id === this.setup.gen.defaultProjectId()) return;
```

Comment:
Why is user resource cleanup is in its CollectionResource class, while project resource cleanup needs to be in the Resource class? (I'm assuming this succeeds in deleting the project)